integrators/ordinary_differential_equations.hpp

```diff
@@ -132,11 +135,15 @@ class FixedStepSizeIntegrator : public Integrator<DifferentialEquation> {
   serialization::FixedStepSizeIntegrator::Kind const kind_;
 };
 
-enum class TerminationCondition {
-  Done,
-  ReachedMaximalStepCount,
-  VanishingStepSize,
-};
+// Any status not listed here is considered "severe" by the |Ephemeris|.
```

Comment:
In fact, this is simply misleading, though not untrue (universal quantitication over the empty set).
This should say that the `Solve` function of an `AdaptiveStepSizeIntegrator` always returns one of the following `Status`es, with the interpretations given.